lesson5.js

```diff
@@ -0,0 +1,31 @@
+const ul = document.getElementById('js-list');
+const info = [
+  {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+  {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+  ];
+
+const fragment = document.createDocumentFragment();
+
+function ListContents(add){
+  for(let i=0; i<=add.length-1; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+    a.textContent = add[i].text;
+    a.href = add[i].to;
+    img.src = add[i].img;
+    img.alt = add[i].alt;
+    fragment.appendChild(li).appendChild(a).insertBefore(img, a.firstChild);
+}
+    ul.appendChild(fragment)
+};
+
```

Comment:
```suggestion
function ListContents(add){
  for(let i=0; i<=add.length-1; i++) {
    const li = document.createElement('li');
    const a = document.createElement('a');
    const img = document.createElement('img');
    a.textContent = add[i].text;
    a.href = add[i].to;
    img.src = add[i].img;
    img.alt = add[i].alt;
    fragment.appendChild(li).appendChild(a).insertBefore(img, a.firstChild);
}
    ul.appendChild(fragment)
};

```
[ IMO ]
I felt that add was not appropriate because it is a verb meaning to add or to add.
I think value, item, list, etc. are good.

Reply:
From now on, we will use "arguments are nouns" and "functions are verbs". Thank you for your advice!